src/fastmcp/client/transports.py

```
@@ -125,7 +125,7 @@ def get_session_id(self) -> str | None:
         """Get the session ID for this transport, if available."""
         return None
 
-    def _set_auth(self, auth: httpx.Auth | Literal["oauth"] | str | None):
+    def _set_auth(self, auth: httpx.Auth | OAuth | Literal["oauth"] | str | None):
         if auth is not None:
             raise ValueError("This transport does not support auth")
```

Ruff (0.14.11)

Avoid specifying long messages outside the exception class

(TRY003)